content/learning-paths/servers-and-cloud-computing/github-on-arm/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title:  Deploy Self-Hosted GitHub Actions Runner on GCP C4A (Arm64) Instance
+ 
+minutes_to_complete: 30
+
+who_is_this_for: This Learning Path is for DevOps engineers, system administrators, or developers who want to deploy GitHub Actions Runner on GCP Arm64 instances.
+
+learning_objectives:
+       - Provision an Arm64 VM on Google Cloud Platform using the Tau C4A instance family.
```

Comment:
Tau is a separate VM family. C4A is Google Axion processor family. Bot are different terminologies.
Simply write : "**Provision an Arm VM on the Google Cloud Platform using the C4A Google Axion instance family.**"